docs/DifferentiableProgramming.md

```diff
@@ -1302,7 +1389,7 @@ Method `move(along:)` will not be synthesized because a default implementation
 already exists.
 
 ```swift
-struct Point<T: Real>: @memberwise Differentiable, @memberwise AdditiveArithmetic {
+struct Point<T: Real>: @memberwise Differentiable, AdditiveArithmetic {
```

Comment:
Would the `@memberwise` attribute apply to both the `Differentiable` conformance and the `AdditiveArithmetic` conformance?

Reply:
This change was unintended. Fixed.